addons/mail/__manifest__.py

```diff
@@ -138,6 +138,7 @@
             'mail/static/src/core/common/**/*',
             'mail/static/src/core/web_portal/**/*',
             'mail/static/src/core/web/**/*',
+            'mail/static/src/chatter/web_portal/**/*',
```

Comment:
Features are defined with `**` instead of hard-coding them (see rule just below). What you are adding is basically a new bundle possibility `web_portal`, for all features.
(Note: it is already done for core above, which is a generic one that must come before all others.)

Suggestion
```suggestion
            'mail/static/src/**/web_portal/**/*',
```
and move the line after `/common/` and before `/web/` to avoid potential dependency issues (`common` code should be loaded before, `web` specific after)